content/installation/net/configuration/YAML-properties-net.md

```diff
@@ -288,6 +289,20 @@ agent:
     enable: false
 ```
 
+### Troubleshooting Options
```

Comment:
We don't *have* to document these settings here. There are pros and cons to documenting these settings here. 

We do need to update this article: https://github.com/Contrast-Security-OSS/docs/blob/master/content/troubleshooting/net/GettingNetAgentLogs.md

Reply:
Ooh, I understand what you were wanting now.

Reply:
Fixed.